controllers/networkfence_controller.go

```diff
@@ -63,6 +63,11 @@ func validateNetworkFenceSpec(nwFence *csiaddonsv1alpha1.NetworkFence) error {
 	if nwFence.Spec.Cidrs == nil {
 		return errors.New("required parameter cidrs is not specified")
 	}
+
+	if nwFence.Spec.FenceState != csiaddonsv1alpha1.Fenced && nwFence.Spec.FenceState != csiaddonsv1alpha1.Unfenced {
+		return errors.New("invalid fence state. fence state can only be either of Fenced or Unfenced")
```

Comment:
Should we log fence state in the error message?

Reply:
As suggested in the comment [1], will be removing this check.


[1] https://github.com/csi-addons/kubernetes-csi-addons/pull/152#discussion_r867643651